src/styles/block/watch.scss

```diff
@@ -1,27 +1,41 @@
 .watch {
   padding: 20px;
-
-  &__hand-second {
-    width: 170px;
-    height: 5px;
-
+  &__hand {
     @include hand-position;
-    right: -15px;
 
     background-color: $main-color;
     border-radius: $hand-border-radius;
-    transform: rotate(90deg);
 
-    animation: time-stak 60s linear infinite;
-    transform-origin: right;
-    animation-timing-function: steps(60, jump-start);
-    animation-delay: 3s;
+    &--second {
+      width: 170px;
+      height: 5px;
+
+      right: -15px;
+
+      transform: rotate(90deg);
```

Comment:
To avoid these weird manipulations with both hands (rotation, transform-origin etc...) you need to have them positioned absolutely.
Then your hands will be both vertical and have common props.